configs/configs_test.go

```diff
@@ -16,30 +16,37 @@ package configs
 
 import (
 	"encoding/json"
+	"fmt"
 	"testing"
 	"time"
 )
 
 func TestJSONDuration(t *testing.T) {
-	var s struct {
-		Dur JSONDuration
+	testCases := []struct {
+		json         []byte
+		wantErr      string
+		wantDuration time.Duration
+	}{
+		{[]byte(`{"dur":"10s"}`), "", time.Second * 10},
```

Comment:
nit: Please use named literals on the struct fields. This way you don't have to specify default/unrelated values as well.